packages/editor/src/components/post-author/index.js

```diff
@@ -97,6 +115,29 @@ function PostAuthor() {
 		return null;
 	}
 
+	if ( ! isCombobox ) {
+		const selectId = 'post-author-selector-' + instanceId;
+
+		return (
+			<PostAuthorCheck>
+				<label htmlFor={ selectId }>{ __( 'Author' ) }</label>
+				<select
```

Comment:
label select... are all abstracted in SelectControl, we should probably use that instead

Reply:
Ah, thanks - forgot about that component. I will use it here.